scripts/lint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+go get golang.org/x/lint/golint
+
+# Http => HTTP change would require a major version bump as all those functions/types are exported
+if [[ -z $(golint | grep -vE "Http.*should be.*HTTP") ]]; then
```

Comment:
I would wrap `$(..)` with `"`. Otherwise you will get syntax errors when the output contains whitespace.

Reply:
Can we rename the types affected to `Http` and set up legacy type aliases?